arm/Microsoft.ServiceBus/namespaces/deploy.bicep

```diff
@@ -244,5 +299,4 @@ module serviceBusNamespace_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignmen
 output serviceBusNamespaceResourceId string = serviceBusNamespace.id
 output serviceBusNamespaceResourceGroup string = resourceGroup().name
 output serviceBusNamespaceName string = serviceBusNamespace.name
-output defaultAuthorizationRuleId string = defaultAuthorizationRuleId
 output serviceBusConnectionString string = 'Endpoint=sb://${serviceBusNamespaceName_var}.servicebus.windows.net/;SharedAccessKeyName=${listkeys(resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', serviceBusNamespaceName_var, 'RootManageSharedAccessKey'), '2017-04-01').primaryKey}'
```

Comment:
Missing output description. Choose to address here or open an issue for it

Reply:
added